Source/WebKit/Platform/IPC/HandleMessage.h

```diff
@@ -123,118 +123,80 @@ void logReply(const Connection& connection, MessageName messageName, const T&...
 #endif
 }
 
-// Dispatch functions with no reply arguments.
-
-template <typename C, typename MF, typename ArgsTuple, size_t... ArgsIndex>
-void callMemberFunctionImpl(C* object, MF function, ArgsTuple&& args, std::index_sequence<ArgsIndex...>)
+template<typename TupleType, typename F>
```

Comment:
Is this related to `std::apply`? Maybe mention in commit msg why that is not usable?

There is a reason why `functor` is a `const T&` instead of `T&&`?

Reply:
In theory this could be `std::apply` ... In practice I'll compare the generated code.

Reply:
`std::apply()` works fine, no difference in generated code.